bundles/org.openhab.binding.tesla/src/main/java/org/openhab/binding/tesla/internal/handler/TeslaVehicleHandler.java

```diff
@@ -283,10 +283,13 @@ public void handleCommand(ChannelUID channelUID, Command command) {
                             }
                         }
                         if (amps != null) {
-                            if (amps < 5 || amps > 32) {
-                                logger.warn("Charging amps can only be set in a range of 5-32A, but not to {}A.", amps);
+                            if (amps > 32) {
+                                logger.warn("Charging amps cannot be set higher than 32A, {}A was requested", amps);
                                 return;
                             }
+                            if (amps < 5) {
+                                logger.warn("Charging amps should be set higher than 5A to avoid excessive losses.");
```

Comment:
I din't think this qualifies as a `WARN` level messages. `WARN` should be used if something goes wrong and the add-on can't handle that. In this case it is more an information, so `INFO` would be the correct level.

Reply:
Done.